plugins/statistics.koplugin/main.lua

```diff
@@ -1897,20 +1897,20 @@ end
 function ReaderStatistics:genResetBookSubItemTable()
     local sub_item_table = {}
     table.insert(sub_item_table, {
-        text = _("Reset statistics per book"),
+        text = _("Reset statistics for current book"),
```

Comment:
Probably worth fixing it to `Reset statistics for the current book` while we're here ;).